sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -37,18 +37,50 @@ This exporter sends traces to the configured Azure Monitor Resource using HTTPS.
 
 Refer to [`Program.cs`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Program.cs) for a complete demo.
 
+### Traces
+
 ```csharp
-using Azure.Monitor.OpenTelemetry.Exporter;
+using System.Diagnostics;
+using OpenTelemetry;
 using OpenTelemetry.Trace;
 
-OpenTelemetry.Sdk.CreateTracerProviderBuilder()
-    .AddSource("Demo.DemoClient")
-    .AddAzureMonitorTraceExporter(o => {
-        o.ConnectionString = "<Your Connection String>";
-    })
+ActivitySource activitySource = new("MyCompany.MyProduct.MyLibrary");
```

Comment:
What's the lifecycle management?

Reply:
My initial intent was to have minimal code samples of how to add the AzureMonitorExporter (owned by this project).

Let me discuss with the team how we want to make best use of the Readme.